<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="Sistema">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Asistencia">
       <attribute name="title">
        <string>Asistencia</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="Ingreso_Alumnos">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="ingreso">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>75</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Ingreso de Alumnos:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="alumnos_layout">
            <item>
             <widget class="QLineEdit" name="input_name">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <underline>false</underline>
               </font>
              </property>
              <property name="placeholderText">
               <string>Nombre y Apellido</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_name">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
                <underline>false</underline>
               </font>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Ingreso_Dias_Clase">
          <item>
           <widget class="QLabel" name="dias_totales">
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>75</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Dias de Clase:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="clases_layout">
            <item>
             <widget class="QSpinBox" name="clases_totales">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <underline>false</underline>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_clases">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
                <underline>false</underline>
               </font>
              </property>
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Asistencia_Alumno">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="asistencia">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Asistencia:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="asistencia_layout">
            <property name="spacing">
             <number>70</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="Clase_Total">
              <item>
               <widget class="QLabel" name="total">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Clases:</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="display_clases">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Presencias">
              <item>
               <widget class="QLabel" name="presentes">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Presencias:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="display_presentes">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Ausencias">
              <item>
               <widget class="QLabel" name="ausentes">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Faltas:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="display_ausentes">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="digitCount">
                 <number>2</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="BotonesAsistencia">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="Add">
                <property name="spacing">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QPushButton" name="add_presentes">
                  <property name="text">
                   <string>Presente</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="add_ausentes">
                  <property name="text">
                   <string>Ausente</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="Remove">
                <item>
                 <widget class="QPushButton" name="remove_presentes">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="remove_ausentes">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Notas">
       <attribute name="title">
        <string>Notas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="alumno_layout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="Alumno">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="text">
             <string>Alumno:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QComboBox" name="alumnos">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <underline>false</underline>
             </font>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="notas_layout">
          <item>
           <widget class="QDoubleSpinBox" name="input_notas">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="send">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Enviar</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="muestreo">
          <item>
           <layout class="QVBoxLayout" name="muestreo_2">
            <property name="leftMargin">
             <number>60</number>
            </property>
            <property name="rightMargin">
             <number>50</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Notas:</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QListWidget" name="lista_notas">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="Promedio">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="leftMargin">
               <number>200</number>
              </property>
              <property name="rightMargin">
               <number>200</number>
              </property>
              <item alignment="Qt::AlignRight">
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Promedio:</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLCDNumber" name="display_promedio">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>150</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="smallDecimalPoint">
                 <bool>true</bool>
                </property>
                <property name="digitCount">
                 <number>3</number>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
                <property name="value" stdset="0">
                 <double>0.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="button_promedio">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Promedio</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlumnos">
    <property name="title">
     <string>Alumnos</string>
    </property>
   </widget>
   <addaction name="menuAlumnos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
